--- thepkg
+++ thepkg-extended-verbose
@@ -0,0 +0,0 @@
-		err "empty or missing manifest file." # [patch: thepkg-extended-verbose]
+		# detect when went wrong
+		# reason: `tar` is ran in pipe
+		# and exit status is lost in POSIX sh
+		# then exit status would be 0
+
+		printf ' x> %s\n' >&2 \
+			"Extracding the pkg went wrong." \
+			"To remove the empty pkg installed," \
+			"  please run: ${0##*/} del '$thepkgname'" \
+		;
+		exit 3
